src/msg/producer/config/writer.go

```diff
@@ -107,6 +107,9 @@ type WriterConfiguration struct {
 	// IgnoreCutoffCutover allows producing writes ignoring cutoff/cutover timestamp.
 	// Must be in sync with AggregatorConfiguration.WritesIgnoreCutoffCutover.
 	IgnoreCutoffCutover bool `yaml:"ignoreCutoffCutover"`
+	// WithoutConsumerScope drops the consumer tag from the metrics. For large m3msg deployments the consumer tag can
+	// add a lot of cardinality to the metrics.
+	WithoutConsumerScope bool `ymal:"withoutConsumerScope"`
```

Comment:
typo: ymal

Reply:
lol how did the linter not catch this (at least locally)